serialization/common/test/IntegrationTest.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2019-2021 JetBrains s.r.o.
+ * Use of this source code is governed by the Apache 2.0 License that can be found in the LICENSE.txt file.
+ */
+
+package kotlinx.datetime.serialization.test
+
+import kotlinx.datetime.*
+import kotlinx.datetime.serializers.*
+import kotlinx.serialization.Contextual
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.modules.SerializersModule
+import kotlinx.serialization.modules.contextual
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class IntegrationTest {
+
+    @Serializable
+    data class Dummy(
+        @Contextual val instant: Instant,
```

Comment:
Great!
Can we have another one, where non-default serializers are specified with an annotation by the class author?